common/src/main/java/com/wynntils/functions/CharacterFunctions.java

```diff
@@ -87,6 +89,45 @@ public Double getValue(FunctionArguments arguments) {
         }
     }
 
+    public static class StatusEffectsFunction extends Function<String> {
+        @Override
+        public String getValue(FunctionArguments arguments) {
+            List<StatusEffect> statusEffectsList = Models.StatusEffect.getStatusEffects();
```

Comment:
You can use streams on both of these so you'll only need 1 list. For the contains function filter it to only include the query and return if it's empty or not

Reply:
Got it. Give me a bit

Reply:
Ok refactored